requirements/pytorch/collect_env_details.py

```diff
@@ -24,7 +24,7 @@
 import torch
```

Comment:
let's keep this at the root of the requirement and, if needed, extend by collecting info needed for LAI?